applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py

```diff
@@ -8,10 +9,6 @@
 from KratosMultiphysics.testing.utilities import ReadDistributedModelPart
 from KratosMultiphysics.TrilinosApplication import trilinos_linear_solver_factory
 
-
-def GetFilePath(fileName):
-    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)

```

Comment:
this should be 
~~~
def GetFullPathToFile(fileName):
    return pathlib.Path(__file__).absolute().parent / fileName
~~~

and so on ...

Reply:
but here it is removed